cmd/notation/logout.go

```diff
@@ -16,8 +16,9 @@ func logoutCommand(opts *logoutOpts) *cobra.Command {
 		opts = &logoutOpts{}
 	}
 	return &cobra.Command{
-		Use:   "logout [server]",
+		Use:   "logout [flags] [server]",
 		Short: "Log out the specified registry hostname",
```

Comment:
```suggestion
		Short: "Log out from the logged in registries",
```
This is stated in the [spec](https://github.com/notaryproject/notation/blob/main/specs/notation-cli.md). Interestingly, this command's Long is shorter than Short.